docs/api.md

```diff
@@ -50,47 +50,46 @@ nos.getAddress()
   .catch((err) => alert(`Error: ${err.message}`));
 ```
 
-## `getBalance`
+### `getBalance`
 The `getBalance` function provides the balance of a certain address for a
 specified asset or NEP5 token.  It does not require the user to grant permission.
 
-### Parameters
-1. `string` - The asset ID or NEP5 token script hash.
-2. `string` (Optional) - The address of the user you'd like to receive the balance for. This defaults to the currently logged on user if the parameter is not passed.
+#### Parameters
+1. `asset` [string] - The asset ID or NEP5 token script hash.
+2. `address` [string] (Optional) - The address of the user you'd like to receive the balance for. This defaults to the currently logged on user if the parameter is not passed.
```

Comment:
I like the way snowy writes the parameters out here for options/config objects, and I'm interested to see how you feel about it.  It'd end up looking something like this:

```
#### Parameters
* `config` [object] - The config options to perform this operation.
* `config.asset` [string] - The asset ID or NEP5 token script hash.
* `config.address` [string] (Optional) - The address of the user you'd like to receive the balance for. This defaults to the currently logged on user if the parameter is not passed.
```

Reply:
Yeah that's a good idea!